package.json

```diff
@@ -1,22 +1,20 @@
 {
 	"name": "time-budgeter",
-	"version": "0.0.1",
+	"version": "0.0.0",
```

Comment:
Match with version from changelog

Reply:
i dont want to maintain the version in package.json and package-lock.json

Reply:
Why not

Reply:
you need to run npm i every time you update the package.json version, and since its not a library it doesnt even matter